resources/DevLog.md

```
@@ -0,0 +1,91 @@
+# DevLog for Sohojogi Project
+
+---
+
+## Week 1: Idea Generation, Background Research, and Pitch Preparation  
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)